WooCommerce/WooCommerceTests/Mockups/MockProductVariation.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+@testable import Yosemite
+
+final class MockProductVariation {
+    func productVariation(siteID: Int64 = 2019,
+                          productID: Int64 = 2020,
+                          variationID: Int64 = 2783) -> ProductVariation {
```

Comment:
I have doubts about these arguments and default values (e.g. `"chocolate.com"`). I feel like it's better to just have an empty default object:

```swift
    func productVariation() -> ProductVariation {
        siteID: 0, // numbers are zero
        permalink: "" // required strings are empty strings
```

And then let the unit test define the values using `.copy()`

```swift
let variation = MockProductVariation().productVariation().copy(siteID: 12, productID: 12, variationID: 3, something: "", anotherThing: "")
```

In this way, I think it's more explicit what data the unit test needs. And we can guarantee that no default data in `productVariation()` is being used. It's just a helper so we don't need to write lots of (unnecessary) default arguments. 

This is just a not well thought out idea. Just sharing in case you have some thoughts. 🙂

Reply:
This is a good point! The code here is from the existing mock from `Yosemite`, but since I've been using `copy` for fixtures it does make sense to just return an empty object with empty default values. will update!

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/2566/commits/deef6afc0e41e5aba543c82ebb8083450509d2fe